docs/user_guide/example.rst

```diff
@@ -0,0 +1,62 @@
+Use case example
+================
+
+Here are two simple example on how you can use the API endpoints to access data. The first example demonstrates how to find all instruments associated with a field, and the second example demonstrates how to find all risers with a specified length and sort them by field.
```

Comment:
Here are two simple example**s** **showing** how you can use the API endpoints to access data.